PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -297,6 +299,21 @@ public String getUserAuthenticationEmail() {
         return this.userAuthenticationEmail;
     }
 
+    /**
+     * Optional: User phone number to initiate a quicker authentication flow in cases where the user
+     * has a PayPal Account with the same phone number.
+     *
+     * @param userPhoneNumber - phone number of the payer
+     */
+    public void setUserPhoneNumber(@Nullable String userPhoneNumber) {
```

Comment:
Should we include some formatting guidelines, for example would `1112223333`, `(111) 222-3333`, `+1 (111)222 - 3333`, etc all be accepted?

Reply:
The Gateway will be stripping out any non-numeric character. But it still might be nice to give merchants some guidance to remove any potential confusion on the format of the phone number.

Maybe we suggest merchants to only pass in numeric characters? `1112223333` What do y'all think?

cc @agedd

Reply:
My two cents - we could either say `any non numeric characters will be removed from the input` to make it easier for merchants so they don't need to do any format changing, or we could just request input in all numeric characters to avoid confusion. Either is good with me